### PR TITLE
minor updates for clarity to VGP assembly tutorial

### PR DESCRIPTION
**Pull request by pickettbd**
This is meant to be applied after the changes from gallardoalba
in PR #3142 dated 21 Jan. 2022. I made the changes before I saw that PR, but I went through and made the same changes as much as possible. My changes are primarily re-wording and grammar for added clarity.

### CODE REVIEW

File: topics/assembly/tutorials/vgp_genome_assembly/tutorial.md [hunk 1]
```diff
@@ -128,22 +132,26 @@ The first step is to get the datasets from Zenodo. The VGP assembly pipeline use
...
+>       - Click `Add Definition` button and select `Collection Name`: column `E`
 >    - Click `Apply` and press <kbd>Upload</kbd>
 >
 {: .hands_on}
-    
+
+
+### HiFi reads preprocessing with **Cutadapt**
+
+Once we have retrieved the data, we need to trim residual adaptor sequences from the PacBio HiFi reads, in order remove the bases that could interfere with the assembly process. For this purpose, we are going to use Cutadapt, an open-source tool written mainly in Python language.
```
Comment (gallardoalba) [inline topics/assembly/tutorials/vgp_genome_assembly/tutorial.md:144]:
```suggestion

```

File: topics/assembly/tutorials/vgp_genome_assembly/tutorial.md [hunk 2]
```diff
@@ -289,22 +297,22 @@ Before proceeding to the next section, we need to carry out some operations on t
...
 >
 > 2. {% tool [Replace](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_find_and_replace/1.1.3) %} with the following parameters:
 >    - {% icon param-file %} *"File to process"*: output file of **Replace** {% icon tool %}
 >    - *"Find pattern"*: `,`
->    - *"Replace all occurences of the pattern"*: `Yes`
+>    - *"Replace all occurrences of the pattern"*: `Yes`
 >    - *"Find and Replace text in"*: `entire line`
 >
-> 3. {% tool [Search in textfiles (grep)](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_grep_tool/1.1.1) %} with the following parameters:
+> 3. {% tool [Search in text files (grep)](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_grep_tool/1.1.1) %} with the following parameters:
```
Comment (gallardoalba) [inline topics/assembly/tutorials/vgp_genome_assembly/tutorial.md:309]:
```suggestion
> 3. {% tool [Search in textfiles (grep)](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_grep_tool/1.1.1) %} with the following parameters:
```

File: topics/assembly/tutorials/vgp_genome_assembly/tutorial.md [hunk 3]
```diff
@@ -997,17 +1004,18 @@ Despite Hi-C generating paired-end reads, we need to map each read separately. T
 > 4. Rename the output as `BAM reverse`
 >
 > 5. {% tool [Filter and merge](toolshed.g2.bx.psu.edu/repos/iuc/bellerophon/bellerophon/1.0+galaxy0) %} with the following parameters:
->    - {% icon param-file %} *"First set of reads"*: `BAM forward` 
+>    - {% icon param-file %} *"First set of reads"*: `BAM forward`
 >    - {% icon param-file %} *"Second set of reads"*: `Bam reverse`
```
Comment (gallardoalba) [inline topics/assembly/tutorials/vgp_genome_assembly/tutorial.md:1008]:
```suggestion
>    - {% icon param-file %} *"Second set of reads"*: `BAM  reverse`
```

File: topics/assembly/tutorials/vgp_genome_assembly/tutorial.md (hunk 1 above)
Comment (gallardoalba) [reply topics/assembly/tutorials/vgp_genome_assembly/tutorial.md:144]: It was duplicated.

File: topics/assembly/tutorials/vgp_genome_assembly/tutorial.md (hunk 2 above)
Comment (gallardoalba) [reply topics/assembly/tutorials/vgp_genome_assembly/tutorial.md:309]: It correspond to the tool name https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_grep_tool/1.1.1

File: topics/assembly/tutorials/vgp_genome_assembly/tutorial.md [hunk 4]
```diff
@@ -289,22 +297,22 @@ Before proceeding to the next section, we need to carry out some operations on t
...
 >
-> 3. {% tool [Search in textfiles (grep)](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_grep_tool/1.1.1) %} with the following parameters:
+> 3. {% tool [Search in text files (grep)](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_grep_tool/1.1.1) %} with the following parameters:
 >    - {% icon param-file %} *"Select lines from"*: output file of the previous step.
 >    - *"Type of regex"*: `Basic`
 >    - *"Regular Expression"*: `Haploid`
 >
 > 4. {% tool [Convert delimiters to TAB](Convert characters1) %} with the following parameters:
->    - {% icon param-file %} *"in Dataset"*: output of **Search in textfiles** {% icon tool %}
+>    - {% icon param-file %} *"in Dataset"*: output of **Search in text files** {% icon tool %}
```
[the remark this answers is not in the document]
Comment (gallardoalba) [reply topics/assembly/tutorials/vgp_genome_assembly/tutorial.md:315]: Same as before.

**Comment by gallardoalba** [comment]
Thank you so much. I included a few modifications in order to fix some incompatibilities with a recent PR submitted by @abueg.

**Review by gallardoalba — APPROVED** [review]

**Comment by bgruening** [comment]
Awesome, thanks a lot @pickettbd!

**Pull request merged**